### PR TITLE
Add several spelling corrections

### PR DESCRIPTION
**Pull request by luzpaz**
@vikivivi please review, TIA!

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -14821,6 +14847,7 @@ extedns->extends
 extemely->extremely
 exten->extent
 extenal->external
+extendability->extendibility
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:14850]: I suggest you drop or stash this. 
`extendability` & `extendibility` are both valid as well as invalid depending on which dictionaries you consulted. Validity of both words are inconsistent across many dictionaries.

Both `extendability` & `extendibility` are valid in hunspell, aoo-mozilla-en-dict
Both `extendability` & `extendibility` are invalid in aspell, chrome browser,

`extendability` is valid in hunspell, aoo-mozilla-en-dict
`extendability` is invalid in aspell, chrome browser, ispell

`extendibility` is valid in hunspell, aoo-mozilla-en-dict, ispell
`extendibility` is invalid in aspell, chrome browser

In source code context, in my opinion, majority actually mean `extensibility`.
`extensibility` is also inconsistent in various dictionaries.

https://www.lexico.com/definition/extendability (Both valid, lexico.com has merge with dictionary.com)
https://www.merriam-webster.com/dictionary/extendability  (Both valid)
https://www.dictionary.com/misspelling?term=extendability  (Both valid)
https://wikidiff.com/extensibility/extendibility  (extendibility vs extensibility)

https://www.ldoceonline.com/spellcheck/english/?q=extendability  (Both invalid)
https://www.oxfordlearnersdictionaries.com/spellcheck/english/?q=extendability  (Both invalid)
https://dictionary.cambridge.org/spellcheck/english/?q=extendibility  (Both invalid)

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -18999,9 +19033,13 @@ intercontinential->intercontinental
 intercontinetal->intercontinental
 interdependant->interdependent
 interecptor->interceptor
+interect->intersect, interact,
+interected->intersected, interacted,
 interecting->intersecting, interacting,
 interection->intersection, interaction,
+interectioned->instersectioned
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:19040]: Suggested word is a misspelling. At the same time `intersectioned` is also a misspell. So you might want to just suggest `intersection`. Anyway, there is only 1 hit.
```text
interectioned->intersection
```
https://github.com/search?q=interectioned&type=code (1)

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -22233,6 +22275,8 @@ neibors->neighbors
 neice->niece, nice,
 neigbhorhood->neighborhood
 neigbhorhoods->neighborhoods
+neigbhour->neighbor
+neigbhours->neighbors
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:22279]: I think you should suggest `en-GB` and `en-US` separately. Try not to mix both `bour` and `bor`
```
neigbhour->neighbour
neigbhours->neighbours
neigbhor->neighbor
neigbhors->neighbors
```
* https://github.com/search?q=neigbhour&type=code  (3K)
* https://github.com/search?q=neigbhours&type=code  (1K)
* https://github.com/search?q=neigbhor&type=code  (3K)
* https://github.com/search?q=neigbhors&type=code  (7K)

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -23472,6 +23520,7 @@ orthoganal->orthogonal
 orthoganalize->orthogonalize
 orthognal->orthogonal
 orthonormalizatin->orthonormalization
+ortogonal->orthogonal
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:23523]:
```text
ortogonality->orthogonality
```
https://github.com/search?q=ortogonality&type=code  (1K)

File: codespell_lib/data/dictionary.txt [hunk 5]
```diff
@@ -30008,6 +30070,7 @@ severley->severely
 severly->severely
 sevice->service
 sevirity->severity
+sevral->several
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:30073]:
```text
sevrally->severally
```
https://github.com/search?q=sevrally&type=code  (8)

File: codespell_lib/data/dictionary.txt [hunk 6]
```diff
@@ -32791,6 +32858,7 @@ techer->teacher
 techers->teachers
 teches->teaches
 techical->technical
+techically->technically
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:32861]: You might want to suggest `tactically`?
```text
techically->technically, tactically,
```

File: codespell_lib/data/dictionary.txt [hunk 7]
```diff
@@ -33738,6 +33810,7 @@ transistion->transition
 transistioning->transitioning
 transistions->transitions
 transitionned->transitioned
+transitionning->transitioning
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:33813]:
```text
transitionnal->transitional
transitionns->transitions
```
* https://github.com/search?q=transitionnal&type=code  (2K)
* https://github.com/search?q=transitionns&type=code  (16)

File: codespell_lib/data/dictionary.txt [hunk 8]
```diff
@@ -33771,6 +33844,12 @@ transocders->transcoders
 transocdes->transcodes
 transocding->transcoding
 transocdings->transcodings
+transofrm->transform
+transofrmation->transformation
+transofrmations->transformations
+transofrmed->transformed
+transofrming->transforming
+transofrms->transforms
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:33852]:
```text
transofrmer->transformer
transofrmers->transformers
```
* https://github.com/search?q=transofrmer&type=code  (930)
* https://github.com/search?q=transofrmers&type=code  (119)

File: codespell_lib/data/dictionary.txt [hunk 9]
```diff
@@ -36398,6 +36479,8 @@ workes->works
 workfow->workflow
 workfows->workflows
 workign->working
+workled->worklet, worked,
+workleds->worklets, worked,
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:36483]: I am not favour of this entry. This one is more like a CamelCase for workLEDs, workLeds. 
Looking at the search results seem valid coding.
Splitting `work` and `LEDs` will also break source code.
Personally, I would drop this one.

* https://github.com/search?q=workled&type=code  (234)
* https://github.com/search?q=workleds&type=code  (10)

**Review by vikivivi — CHANGES_REQUESTED** [review]

**Comment by luzpaz** [comment]
Thanks. I've added your suggestions with no objection.

**Review by vikivivi — APPROVED** [review]
LGTM

**Pull request merged**